lib/config/config-initializer.js

```diff
@@ -76,12 +80,24 @@ function installModules(config) {
     if (modules.length === 0) {
         return;
     }
+
+    // Add eslint to list in case user does not have it installed locally
+    modules.unshift("eslint");
```

Comment:
Put at the beginning of the array for clarity, so that you see it's installing right after reading the `"Local ESLint installation not found."` message